web/modules/custom/collection/src/ContextProvider/CollectionRouteContext.php

```diff
@@ -48,6 +51,22 @@ public function getRuntimeContexts(array $unqualified_context_ids) {
         }
       }
     }
+    elseif ($route_parameters = $this->routeMatch->getParameters()) {
+      /** @var ContentEntityInterface $route_parameter */
+      foreach ($route_parameters as $route_parameter) {
```

Comment:
Well yes but again, there is no "clear" documentation on the pages we want to show it where we could just list the routes. 
Also, currently, we do not have something like <entity_type>/parameter/<something else>/<other parameter> or something similar. 
Is there a better way that you suggest? The way I would see that, if this is any route path of an entity then the context can be available. If later on we create more complicated functionality and it breaks our tests, then we can fix this. (no?)